docs/content/node-operation/spork.mdx

```diff
@@ -1,22 +1,97 @@
 ---
 title: Spork Process
-description: Information about the sporking process
+description: What to do during a spork
 ---
 
-## What is a Spork?
+## Overview
 
-In the mature Flow, new nodes can be staked and un-staked as the protocol advances from epoch to epoch. However, we are not there yet.
-Hence, currently every couple of weeks we turn-off the network, update the identity list to include (and exclude) nodes and then turn the network back on again.
-We call this process a **Spork**.
+A spork is a coordinated network upgrade process where node operators upgrade their node software and
+re-initialize with a consolidated representation of the previous spork's state. This enables rapid development
+on the Flow Protocol and minimizes the impact of breaking changes. 
 
-Also, as Flow evolves, we are continuously adding new features and discovering and fixing bugs in the Flow node software.
-We also utilize a Spork as an opportunity to update the nodes with the latest releases.
+The Flow network sporks approximately once every two months.  Upcoming sporks
+are announced in advance on the `#flow-validators-announcements` **Discord** channel
+and in [Upcoming Sporks](/node-operation/upcoming-sporks). The `#flow-validators-announcements` channel is 
+also used to coordinate during the spork process.
 
-## How frequently do we Spork?
+This guide is for existing operators participating in a spork. See [Node Bootstrap](/node-operation/node-bootstrap)
+for a guide to joining the network for the first time. 
 
-The Flow network sporks approximately once per month. Upcoming sporks are announced in advance on the `#flow-validators-announcements` **Discord** channel.
+## Step 1 - Cleaning Up Previous Spork State
 
-## Past and upcoming Sporks
+Once the spork start has been announced on, stop your node and clear your database. The node should stay stopped for the duration of the spork.
 
-- [Past Mainnet & Testnet Sporks](/node-operation/past-sporks)
-- [Upcoming Sporks](/node-operation/upcoming-sporks)
+<Callout type="warning">
+You can skip this step if it is your first time running a node on Flow.
+</Callout>
+
+1. Stop your Flow node
+2. Clear the contents of your `data` directory that you have previously created. The default location is `/var/flow/data`. The `data` directory contains the Flow chain state.
+
+<Callout type="warning">
+
+If you had a previous node running on **mainnet 16**, you'll need to turn it
```

Comment:
This callout is a bit unclear to me. Can we be more specific than with the timing **_just before joining_** ? How long does the node need to be turned off for ?